databases/mysql8/Portfile

```diff
@@ -49,7 +49,14 @@ if {$subport eq $name} {
                         sha256  9a2c2819310839ea373f42d69e733c339b4e9a19deab6bfec448281554aa4dbb \
                         size    111710205
 
+    depends_build-append \
+                        port:openssl \
+                        port:cyrus-sasl2 \
+                        port:icu \
+                        port:zlib
```

Comment:
So two things.

- Don't use `port:openssl ` instead use `path:lib/libssl.dylib:openssl` in this case, to allow one of the number of options to statisfy this dep.

- ports listed in `depends_lib` do not need also listing in `depends_build`. As all these are already in `depends_build` please just remove completely the `depends_build-append` setting.

Reply:
OK, makes sense completely.  Thank you for your insight, fixing now.

Reply:
Thanks. Sorry if my first comment made you think I was asking for something to be changed. I wasn't, as I thought what you had actually done was correct. It was just the comment about it that was not quite right and I just wanted to confirm what you had actually changed was what you wanted.

Reply:
No worries, my fault for misunderstanding. Thanks.